packages/phasor/src/elements/surface-element.ts

```diff
@@ -4,14 +4,21 @@ import type * as Y from 'yjs';
 import type { Renderer } from '../renderer.js';
 import type { SurfaceManager } from '../surface.js';
 import { isPointIn } from '../utils/hit-utils.js';
-import { deserializeXYWH, type SerializedXYWH } from '../utils/xywh.js';
+import {
+  deserializeXYWH,
+  type SerializedXYWH,
+  type XYWH,
+} from '../utils/xywh.js';
 
 export interface ISurfaceElement {
   id: string;
   type: string;
   xywh: SerializedXYWH;
   index: string;
   seed: number;
+  rotate: number;
```

Comment:
These three properties might not be necessary for certain elements such as connectors and brushes. Perhaps after operations like rotation and flipping, it would be sufficient to just update the corresponding point coordinates.

Reply:
I think it would be better to add a matrix field.

Reply:
> These three properties might not be necessary for certain elements such as connectors and brushes

The last will be deleted for special types.